docs/cosmos-hub/gaiacli.md

```diff
@@ -2,27 +2,41 @@
 
 ## Gaia CLI
 
-::: tip Note
-If you receive this error message:
+`gaiacli` is the tool that enables you to interact with the node that runs on the Cosmos Hub network, whether you run it yourself or not. Let us set it up properly. In order to install it, follow the [installation procedure](./installation.md).
+
+### Setting up gaiacli 
+
+The main command used to set up `gaiacli` is the following:
 
 ```bash
-Must specify these options: --chain-id  when --trust-node is false
+gaiacli config <flag> <value>
 ```
 
-you must choose whether you wish to verify lite client proofs. If you trust the node which you are querying, you can simply pass `--trust-node=true` - otherwise you'll need to specify `--chain-id`.
-:::
+It allows you to set a default value for each given flag. 
 
-`gaiacli` is the command line interface to manage accounts and transactions on Cosmos testnets.
-Its configuration file resides in `$HOME/.gaiacli/config/config.toml` and can be edited either
-by hand or via the `gaiacli config` command:
+First, set up the address of the full-node you want to connect to:
 
 ```bash
-gaiacli config chain-id gaia-9004
+gaiacli config node <host>:<port
+
+// example: gaiacli config node https://77.87.106.33:26657
 ```
 
-For more information on the command usage, refer to its help screen: `gaiacli config --help`.
+If you run your own full-node, just use `tcp://localhost:26657` as the address. 
+
+Then, let us set the default value of the `--trust-node` flag:
 
-Here is a list of useful `gaiacli` commands, including usage examples.
+```bash
+gaiacli config trust-node false
+
+// Set to true if you run a light-client node, false otherwise
```

Comment:
Is this actually true? I don't think `false` currently works.

Reply:
I thought `false` worked for account query? but yeah this is at best partially true